test/testdefs.jl

```diff
@@ -1,10 +1,28 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-using Base.Test
-
-function runtests(name)
+function runtests(name, isolate=true)
+    if isolate
+        mod_name = Symbol("TestMain_", basename(name))
+        m = Module(mod_name, true)
```

Comment:
It's basically an internal function (Although PyCall uses it [last time I checked](https://github.com/JuliaLang/julia/issues/11395)). I'm trying to measure the difference in memory usage now. If there isn't a big difference (likely) I'll probably switch to the non-parentless version below.